examples/kubeval/README.md

```diff
@@ -0,0 +1,32 @@
+# Kubeval
+
+The `kubeval` KRM config function validates configuration using kubeval. This
```

Comment:
What's kubeval? Why does the user need to care? Provide context, don't assume the reader is familiar with all these proper nouns in the catalog.

Reply:
Updated the description